source/v4/posts.yaml

```diff
@@ -70,6 +70,31 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+    delete:
+      tags:
+        - posts
+      summary: Deletes a post
+      description: |
+        Deletes a single post.
+        ##### Permissions
+        Must be logged in as the user or have `manage_system` permission.
```

Comment:
I think it's actually "logged in as the user who created the post or have the `delete_others_posts` permission." right?